scripts/Makefile.am

```diff
@@ -16,12 +16,14 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 scriptsdir = $(libexecdir)/$(PACKAGE_NAME)
-dist_scripts_SCRIPTS = run-anaconda anaconda-pre-log-gen log-capture start-module apply-updates
+dist_scripts_SCRIPTS = run-anaconda anaconda-pre-log-gen log-capture start-module apply-updates 
 
 dist_noinst_SCRIPTS  = makeupdates makebumpver
 
-dist_bin_SCRIPTS = anaconda-cleanup instperf anaconda-disable-nm-ibft-plugin \
-                   anaconda-nm-disable-autocons
+dist_bin_SCRIPTS     = anaconda-cleanup instperf anaconda-disable-nm-ibft-plugin \
+                       anaconda-nm-disable-autocons
```

Comment:
Unrelated whitespace change.

Reply:
This one is bad? It aligns correctly the equal sign with the other variables in the Makefile.am 🤔

Reply:
It's not bad, the issue @KKoukiou is pointing out is that the change is side effect of the commit purposed to do something else. Feel free to add another commit or PR to fix this.

Reply:
Done, @KKoukiou!